### PR TITLE
styles: renamed colors to match Tailwind CSS 3.0.0

### PR DESCRIPTION
**Pull request by Addin**
This commit uses the colors in our **tailwind.config.js** to match our design in Figma and adds a new custom property.

- **Use text-quaternary instead of text-coolGray-400.**
- **Use text-secondary instead of text-coolGray-300.**
- **Use text-primary instead of text-coolGray-50.**

Before:

image

After:

image

### CODE REVIEW

**Comment by agusmar** [comment]
@Addin What do you think about using primary secondary and tertiary text colors defined in tailwind.config.js for this component?

**Comment by Addin** [comment]
What's the difference between the gray color in our custom config in the tailwind.config.js file and the color palette that Tailwind CSS offers? 👀 it is the same color, right? 
image

**Comment by agusmar** [comment]
If you reference in your component the colors defined in the theme, then you can change them easily whenever you need to, because they work as variables.

But if you just add Tailwind classes in every component and need to change them, you will have to go through every file in the project and change every class manually.

**Comment by SebasEC96** [comment]
Testing

File: src/components/EventPreview/index.tsx [hunk 1]
```diff
@@ -91,12 +91,12 @@ const EventPreview: React.FC<EventPreviewProps> = ({ event, past = false }) => {
...
             hrs
-            <span className="inline-block text-xs font-light text-coolGray-400">
+            <span className="inline-block text-xs font-light text-gray-400">
               Horario en tu ubicación actual
             </span>
           </Card.Paragraph>
         )}
-        <div className="text-coolGray-300">
+        <div className="text-gray-300">
           <BlockContent blocks={event.description} />
```
Comment (SebasEC96) [inline src/components/EventPreview/index.tsx:100]: Test

**Comment by SebasEC96** [comment]
Test 2

**Comment by SebasEC96** [comment]
test v3

**Review by agusmar — APPROVED** [review]
Good job @Addin ! 
Thanks!

**Pull request merged**